### PR TITLE
Effective Volto

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: docs/effective-volto/deployment/simple.md [hunk 1]
```diff
@@ -1 +1,75 @@
...
+
+## Reverse proxies
+
+You need to make available to your users both Volto and the API in public URLs. It's heavily recommended you serve both from the same DNS name, eg. Volto-> `https://mywebsite.com` and API-> `https://mywebsite.com/api` in order to avoid any CORS problem.
+
+```{warning}
+Avoid dealing with CORS in production at all costs. Period.
+```
+
+For SSL support is recommended to use a reverse proxy of your choice that points to Volto port and an API rewrite eg. `/api` in your server. This is the nginx configuration:
```
Comment (tiberiuichim) [inline docs/effective-volto/deployment/simple.md:48]: Is this still true?

File: docs/effective-volto/addons/how-an-add-on-works.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+---
+myst:
+  html_meta:
+    "description": "How does a Volto add-on works?"
```
Comment (stevepiercy) [inline docs/effective-volto/addons/how-an-add-on-works.md:4]: Remove the extra "s" from all the topmatter entries and first heading.

```suggestion
    "description": "How does a Volto add-on work?"
```

File: docs/effective-volto/addons/theme.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
+---
+myst:
+  html_meta:
+    "description": "Create a theme add-on"
+    "property=og:description": "Create a theme add-on"
+    "property=og:title":  "Create a theme add-on"
+    "keywords": "Volto, Plone, Semantic-UI, CSS, Volto theme"
```
Comment (stevepiercy) [inline docs/effective-volto/addons/theme.md:7]:
```suggestion
    "keywords": "Volto, Plone, Semantic UI, CSS, Volto theme"
```

File: docs/effective-volto/addons/dependencies.md [hunk 4]
```diff
@@ -1,3 +1,12 @@
+---
+myst:
+  html_meta:
+    "description": "Add-on dependencies"
+    "property=og:description": "Add-on dependencies"
+    "property=og:title":  "Add-on dependencies"
+    "keywords": "Volto, Plone, Volto add-ons, Javascript, Javascript dependencies"
```
Comment (stevepiercy) [inline docs/effective-volto/addons/dependencies.md:7]:
```suggestion
    "keywords": "Volto, Plone, Volto add-ons, JavaScript, JavaScript dependencies"
```

File: docs/effective-volto/addons/semanticui.md [hunk 5]
```diff
@@ -1,3 +1,12 @@
+---
+myst:
+  html_meta:
+    "description": "Extending SemanticUI"
+    "property=og:description": "Extending SemanticUI"
+    "property=og:title":  "Extending SemanticUI"
+    "keywords": "Volto, Plone, Semantic-UI, CSS, Volto theme"
```
Comment (stevepiercy) [inline docs/effective-volto/addons/semanticui.md:7]:
```suggestion
    "keywords": "Volto, Plone, Semantic UI, CSS, Volto theme"
```

File: docs/effective-volto/addons/semanticui.md [hunk 6]
```diff
@@ -1,3 +1,12 @@
+---
+myst:
+  html_meta:
+    "description": "Extending SemanticUI"
```
Comment (stevepiercy) [inline docs/effective-volto/addons/semanticui.md:4]:
```suggestion
    "description": "Extending Semantic UI"
```

File: docs/effective-volto/addons/semanticui.md [hunk 7]
```diff
@@ -1,3 +1,12 @@
...
+    "property=og:description": "Extending SemanticUI"
+    "property=og:title":  "Extending SemanticUI"
+    "keywords": "Volto, Plone, Semantic-UI, CSS, Volto theme"
+---
+
 # Extending SemanticUI
```
Comment (stevepiercy) [inline docs/effective-volto/addons/semanticui.md:10]:
```suggestion
# Extending Semantic UI
```

File: docs/effective-volto/addons/semanticui.md [hunk 8]
```diff
@@ -1,3 +1,12 @@
+---
+myst:
+  html_meta:
+    "description": "Extending SemanticUI"
+    "property=og:description": "Extending SemanticUI"
```
Comment (stevepiercy) [inline docs/effective-volto/addons/semanticui.md:5]:
```suggestion
    "property=og:description": "Extending Semantic UI"
```

File: docs/effective-volto/addons/semanticui.md [hunk 9]
```diff
@@ -1,3 +1,12 @@
...
+    "property=og:title":  "Extending SemanticUI"
```
Comment (stevepiercy) [inline docs/effective-volto/addons/semanticui.md:6]:
```suggestion
    "property=og:title":  "Extending Semantic UI"
```

File: docs/effective-volto/addons/use-repo-addon.md [hunk 10]
```diff
@@ -1,3 +1,12 @@
+---
+myst:
+  html_meta:
+    "description": "Configure a non-released add-on from an existing repository"
```
Comment (stevepiercy) [inline docs/effective-volto/addons/use-repo-addon.md:4]:
```suggestion
    "description": "Configure an unreleased add-on from an existing repository"
```

File: docs/effective-volto/addons/use-repo-addon.md [hunk 11]
```diff
@@ -1,3 +1,12 @@
...
+    "property=og:description": "Configure a non-released add-on from an existing repository"
```
Comment (stevepiercy) [inline docs/effective-volto/addons/use-repo-addon.md:5]:
```suggestion
    "property=og:description": "Configure an unreleased add-on from an existing repository"
```

File: docs/effective-volto/addons/use-repo-addon.md [hunk 12]
```diff
@@ -1,3 +1,12 @@
...
+    "property=og:title":  "Configure a non-released add-on from an existing repository"
```
Comment (stevepiercy) [inline docs/effective-volto/addons/use-repo-addon.md:6]:
```suggestion
    "property=og:title":  "Configure an unreleased add-on from an existing repository"
```

File: docs/effective-volto/addons/use-repo-addon.md [hunk 13]
```diff
@@ -1,3 +1,12 @@
...
+    "keywords": "Volto, Plone, Volto add-on"
+---
+
 # Configure a non-released add-on from an existing repository
```
Comment (stevepiercy) [inline docs/effective-volto/addons/use-repo-addon.md:10]:
```suggestion
# Configure an unreleased add-on from an existing repository
```

File: docs/effective-volto/addons/what-is-an-addon.md [hunk 14]
```diff
@@ -1,3 +1,12 @@
+---
+myst:
+  html_meta:
+    "description": "What is a Volto addon"
```
Comment (stevepiercy) [inline docs/effective-volto/addons/what-is-an-addon.md:4]:
```suggestion
    "description": "What is a Volto add-on"
```

File: docs/effective-volto/addons/what-is-an-addon.md [hunk 15]
```diff
@@ -1,3 +1,12 @@
...
+    "property=og:description": "What is a Volto addon"
```
Comment (stevepiercy) [inline docs/effective-volto/addons/what-is-an-addon.md:5]:
```suggestion
    "property=og:description": "What is a Volto add-on"
```

File: docs/effective-volto/addons/what-is-an-addon.md [hunk 16]
```diff
@@ -1,3 +1,12 @@
...
+    "property=og:title":  "What is a Volto addon"
```
Comment (stevepiercy) [inline docs/effective-volto/addons/what-is-an-addon.md:6]:
```suggestion
    "property=og:title":  "What is a Volto add-on"
```

File: docs/effective-volto/addons/what-is-an-addon.md [hunk 17]
```diff
@@ -1,3 +1,12 @@
...
+    "keywords": "Volto, Plone, Webpack, Volto add-on"
+---
+
 # What is a Volto addon
```
Comment (stevepiercy) [inline docs/effective-volto/addons/what-is-an-addon.md:10]:
```suggestion
# What is a Volto add-on
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Thank you for doing this! I hope more trainers give their topmatter some search engine love.

I made some nitpicky details suggestions regarding proper names.

Also there is an extra space everywhere that is easier to do a global find and replace than to create lots of demoralizing review comments.

```diff
- "property=og:title":  "
+ "property=og:title": "
```

**Comment by tiberiuichim** [comment]
@stevepiercy Many thanks for the review! I'll apply your suggestions

**Review by stevepiercy — CHANGES_REQUESTED** [review]
A few more nitpicky details that are easier to fix with a global find and replace than a bunch of suggestions.

```diff
- "property=og:title":  "
+ "property=og:title": "
```

File: docs/effective-volto/about_effective_volto.md [hunk 18]
```diff
@@ -0,0 +1,37 @@
+---
+html_meta:
+  "description": ""
+  "property=og:description": ""
+  "property=og:title": ""
+  "keywords": ""
```
Comment (stevepiercy) [inline docs/effective-volto/about_effective_volto.md:6]: Please add MyST topmatter. Also make sure that `myst` is the root key wherever you create a new page and the remaining keys are indented appropriately.

```
myst:
  html_meta:
    "description": "About Effective Volto"
    "property=og:description": "About Effective Volto"
    "property=og:title": "About Effective Volto"
    "keywords": "Volto, Tiberiu Ichim, Víctor Fernández de Alba"
```

**Review by stevepiercy — APPROVED** [review]
I won't have time to review this entire epic training before tomorrow, and I trust you did your best, so I approve. Let's make sure that no new warnings or errors are introduced—specifically in MyST topmatter—then clean it up during the training. Please encourage participants to submit pull requests. Thank you!

**Pull request merged**